lexibank_savelyevturkic.py

```diff
@@ -53,19 +54,34 @@ def cmd_makecldf(self, args):
             id_factory=lambda x: x.id.split('-')[-1]+"_"+slug(x.english),
             lookup_factory='Name')
 
-        wl = Wordlist(str(self.raw_dir.joinpath('turkic_alignment.tsv')))
-        for idx in progressbar(wl):
+        for row in progressbar(self.raw_dir.read_csv(DATAFILE, delimiter="\t", dicts=True)):
+            if row['ID'].startswith("#"):
+                # skip lingpy stuff
+                continue
+            
+            # patch two weird/broken entries:
+            if row['ID'] == '7560':
+                row['TOKENS'] = 'ɕ i v ɘ ʨ'
+            
+            if row['ID'] == '8367':
+                row['ENTRY'] = 'ʒɯl'
+                
+            segs = [segments.get(x, x) for x in row['TOKENS']]
```

Comment:
That's it: `row['TOKENS'].split()`, that's what you need to get to a perfect 100% :)